awx/ui_next/src/screens/Job/JobOutput/HostEventModal.jsx

```diff
@@ -97,7 +97,9 @@ const processCodeMirrorValue = value => {
 };
 
 const processStdOutValue = hostEvent => {
-  const { taskAction, res } = hostEvent.event_data;
+  const {
+    event_data: { taskAction, res },
```

Comment:
`event_data` can be completely missing when things aren't going well on the server